src/protocol/jdbc/org/apache/jmeter/protocol/jdbc/config/DataSourceElement.java

```diff
@@ -289,6 +292,16 @@ private BasicDataSource initPool(String maxPool) {
         }
 
         log.debug("PoolConfiguration:{}", this.dataSource);
+
+        if(isPreinit()) {
+            // side effect - connection pool init - that is what we want
+            try {
+                dataSource.getConnection().close();
+            } catch (SQLException e) {
+                // TODO Auto-generated catch block
+                e.printStackTrace();
```

Comment:
e.printStackTrace() must be avoided and replaced by logging or in this case make it silent ?

Reply:
thanks, I changed it